dist/common/scripts/scylla_util.py

```diff
@@ -99,6 +99,17 @@ def scyllabindir():
 def sysconfdir():
     return str(sysconfdir_p())
 
+def configure_aio_slots():
+    required_aio_slots = str(cpu_count() * 11026 + 65536)
```

Comment:
You could move this calculation into a function, and call it from `scylla_prepare`, which already imports `scylla_util.py`.

Reply:
Good point